tron/node.py

```diff
@@ -464,7 +463,7 @@ def _connect(self):
         #  1. Transport is created (our client creator does this)
         #  2. Our transport is secure, and we can create our connection
         #  3. The connection service is started, so we can use it
-
+        from twisted.internet import reactor
```

Comment:
What's up with this? Was this required to get the tests to pass? do you know why?

Reply:
basically now since action.py depends on node.py (because it validates node_pool being in node pool repo, which we will eventually refactor) and node.py depends on twisted reacotr, it was conflicting with another bit of code that was setting up the reactor. i've removed our own reactor setup as in newer version of twisted it already does the right thing, but there are still remnants of me trying to shuffle code around to keep it from breaking. i'll reduce this diff eventually if we decide to merge this.